sunpy/visualization/imageanimator.py

```diff
@@ -936,16 +936,16 @@ class ImageAnimatorWCS(ImageAnimator):
     Extra keywords are passed to imshow.
 
     """
-    def __init__(self, data, wcs=None, image_axes=[-1, -2], unit_x_axis=None, unit_y_axis=None,
-                 axis_ranges=None, **kwargs):
+    def __init__(self, data, wcs=None, image_axes=[-2, -1], unit_x_axis=None, unit_y_axis=None,
```

Comment:
What is the purpose of slice_axes here?  Is it the dimensions of the wcs object to use in making the axes?